modules/sbo-categories-of-exposable-binding-data.adoc

```diff
@@ -0,0 +1,301 @@
+[id="sbo-categories-of-exposable-binding-data_{context}"]
+= Categories of exposable binding data
+
+{servicebinding-title} provides the ability to expose the binding data values from the backing service resources and custom resource definitions (CRDs).
+
+This section provides examples to show how you can use the various categories of exposable binding data.
+
+
+== Exposing a string from a resource
+.Example
+[source,yaml]
+----
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: hippo
+  namespace: my-postgresql
+  annotations:
+    service.binding/username: path={.metadata.name}
+    ...
+----
+
+In the previous example the string from the `metadata.name` field of `PostgresCluster` custom resource (CR)  is exposed as a username.
```

Comment:
```suggestion
The following example shows how to expose the string from the `metadata.name` field of the `PostgresCluster` custom resource (CR) as a username:
```
Consider moving this sentence earlier to make it an intro to the code sample so that the user knows what they're seeing in the sample. 

If you move it, edit appropriately to make it an introductory sentence and to avoid the passive voice, per my suggestions.